### DISCUSSION TITLE
[2021.08.31] 기능 개발

### DISCUSSION THREAD

**Discussion by leechang4** [pyunji/Hyundai-ITnE-Team_QuadCore#85] [Algorithm]
[기능개발](https://programmers.co.kr/learn/courses/30/lessons/42586)

**Comment by pyunji** [comment]
[기능 개발 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/PR/PR_42586.java)

**Comment by hyk339** [comment]
[기능 개발 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/programmers/PR_42586.java)

**Comment by chaeyeon4u** [comment]
[기능 개발 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/PR_42586.java)

**Comment by leechang4** [comment]
[기능 개발 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/lch/Programmers/PR_42586.java)
